client/components/resource-details/ResourceInfo.tsx

```diff
@@ -1,10 +1,12 @@
 import { useRouter } from 'next/router'
 import styled from 'styled-components'
-import { colours, fontWeight, PageTitle, width, largerThan } from '../../styles'
+import { colours, fontWeight, PageTitle, largerThan, smallerThan, width } from '../../styles'
 import { Resource } from '../../utils'
-import { Review } from '../../utils/types'
+import { Review, Faculty, Status } from '../../utils/types'
 import { TagBubble } from '../common/TagBubble'
+import ReactStars from 'react-stars'
 import ImageGallery from 'react-image-gallery'
+import { formatRelative } from 'date-fns'
```

Comment:
What does this function do? Do we need an entire package just for this? Can we re-implement it ourselves?

Reply:
The library is a date util library like Moment. See the deleted `timeSince` function below for why we probably shouldn't implement it ourselves -- lots of edge cases. We will have much better correctness guarantees from using a maintained third-party library that undergoes its own testing.